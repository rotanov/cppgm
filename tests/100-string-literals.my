identifier u7
identifier u9
identifier r
identifier t
identifier r
identifier Q
identifier T
eof
