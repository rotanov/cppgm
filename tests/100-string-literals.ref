literal "" "foo" "bar" "baz" array of 10 char 666F6F62617262617A00
identifier u7
literal "foo" u8"bar" array of 7 char 666F6F62617200
identifier u9
literal "baz" array of 4 char 62617A00
identifier r
literal "abc" u"abc" array of 7 char16_t 6100620063006100620063000000
identifier t
literal "abc" array of 4 char 61626300
identifier r
invalid "abc"u "abc"t "abc"
identifier Q
literal "(abc)" R"(abc)" array of 9 char 286162632961626300
identifier T
invalid "(abc)" U"ewe" L"moo"
eof
