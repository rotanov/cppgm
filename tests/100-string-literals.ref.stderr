ERROR: ud_suffix does not start with _: "abc"u
ERROR: mismatched encoding prefix in string literal sequence
